gui/src/common/const-string.json

```diff
@@ -17,5 +17,6 @@
   "throughput": "throughput",
   "latency": "latency",
   "iops": "iops",
-  "alert": "alert"
+  "alert": "alert",
+  "lyve_pilot": "lyve_pilot"
```

Comment:
the purpose of const string is getting bloated up. with JSON heirarchy we need to define.

Reply:
List of all the features added inside features to maintain heirarchy.